Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45449&attempt_id=66d309b5-b617-4b34-9e5f-9291b0ebd3a0)

Attempted: patchelfUnstable

<details><summary>Partial log (click to expand)</summary><p>

```
const char __invoke_dynamic_linker__[] __attribute__ ((section (".interp"))) = "/foo/bar";
                                                                ^
1 error generated.
make[2]: *** [Makefile:494: foo.o] Error 1
make[2]: Leaving directory '/private/tmp/nix-build-patchelf-0.10-pre-20180509.drv-0/source/tests'
make[1]: *** [Makefile:710: check-am] Error 2
make[1]: Leaving directory '/private/tmp/nix-build-patchelf-0.10-pre-20180509.drv-0/source/tests'
make: *** [Makefile:436: check-recursive] Error 1
builder for '/nix/store/l1pi72y5gipxdsfv16ij9fh2rmmzkd15-patchelf-0.10-pre-20180509.drv' failed with exit code 2
error: build of '/nix/store/l1pi72y5gipxdsfv16ij9fh2rmmzkd15-patchelf-0.10-pre-20180509.drv' failed
```
</p></details>

--- a/pkgs/development/tools/misc/patchelf/unstable.nix
+++ b/pkgs/development/tools/misc/patchelf/unstable.nix
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoreconfHook ];
   buildInputs = [ ];
 
-  doCheck = true;
+  doCheck = !stdenv.isDarwin;
 
   meta = {
     homepage = https://nixos.org/patchelf.html;

